policy/score_calculator.go

```diff
@@ -113,13 +113,17 @@ func AddDataScore(calculator ScoreCalculator, totalDeps int, finishedDeps int) {
 
 func (c *averageScoreCalculator) Add(score *Score, impact *explorer.Impact) {
 	switch score.Type {
-	case ScoreType_Skip:
+	case ScoreType_Skip, ScoreType_Disabled, ScoreType_OutOfScope:
```

Comment:
if the query is disabled, why would it even be executed and end up with a score?